src/components/common/inside/forms/textInput.css

```
@@ -0,0 +1,30 @@
+
+.target--input-title {
+  color: black;
+  font-size: 0.6em;
+  letter-spacing: 0.09em;
+  margin: 1em 0 0.2em 0;
+  text-align: center;
+  font-family: OpenSansBold;
```

It's a good practice to add more fonts in case your `OpenSansBold` does not load
```suggestion
  font-family: OpenSansBold, Roboto, sans-serif;
```

---

Got it. Thanks